hexa/pipelines/migrations/0048_pipelineversion_unique_pipeline_version_name.py

```diff
@@ -0,0 +1,16 @@
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("pipelines", "0047_pipelineversion_rename_duplicate_pipeline_version_name")
```

Comment:
It is recommended to not mix DDL and DML operations in one migration module 
- https://plotnikova1111.medium.com/django-migrations-tips-89f4e588a005
- https://docs.djangoproject.com/en/5.1/ref/migration-operations/#runpython

Reply:
Reason why I splitted it in 2